code/Dependencies.md

```diff
@@ -38,3 +38,34 @@ To add a new dependency evaluate the following things:
 * Check how much it affects `storj/uplink` size.
 
 Note: many of these are fixable with communication or making our own PR-s.
+
+## Finding the changed code in dependencies
+
+The easiest way to find the minimal set of code that needs to be reviewed is to:
+
+``` sh
+# create a temporary git branch
+git switch -c deps
+
+# vendor direct and indirect dependencies
+go mod vendor
```

Comment:
If all our packages' `go.mod` files are converted to 1.17 and later, can we simplify this to inspect `go.mod` only as it only contains direct+indirect dependencies that end up getting used?

Reply:
However, it might be beneficial to vendor these packages to review/lint them in place, so I think we can leave this as it is.

Reply:
`go.mod` only references the modules not the specific packages. So the vendor can contain less code than modules. But, I'll add it as a separate note.

Reply:
I tried to clarify.